.github/workflows/commitlint.yml

```diff
@@ -0,0 +1,19 @@
+name: commitlint check
```

Comment:
A better workflow name would be nice (one that describes what it does, smth like `Lint commit msgs`)

Reply:
Because this is shown in the Action tab